### DISCUSSION TITLE
Missing Meeting...?

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1601] [Q&A]
Merry Chistma-Kwanza-kkah (& Festivus for the rest of us) everyone! I need help. I have a meeting, _Women 2 Women - Midtown_, that for whatever reason I cannot get to populate in the meeting guide app. It is showing on our site. The plug-in is updated and I've ensured the the permalink is different for both meetings. The locations are different and I've even searched for the meeting just using the location _Chabad Intown_ and only 1 other meeting is showing there. 

Here's the link to the meeting on our site [](https://atlantaaa.org/meetings/?type=in-person&search=women+2+women&meeting=women-2-women-midtown)

**Comment by ginslo** [comment]
Hi @ATLCOTech , I think I found the meeting you're referring to here: [https://atlantaaa.org/meetings/?meeting=women-2-women-midtown ](https://atlantaaa.org/meetings/?meeting=women-2-women-midtown ) and took a cursory look at it. It's in your json feed and I don't see any reason why it wouldn't be picked up by the Meeting Guide, but I could be missing something so will need to defer this to someone with more experience in this area.

**Reply by joshreisner** [reply]
[here are the other reasons](https://meetingguide.helpdocs.io/article/yw9hioe5hl-meeting-updates-not-visible) why a meeting might not show up in the app

did you contact app support? they can be reached at appsupport@aa.org

**Comment by SIA-WEBSUP** [comment]
@ATLCOTech, I did a quick check to see who is providing the info for the one meeting that does appear in the app for that location and it appears to be Georgia State Service Assembly. If that isn't you, then the issue may have more to do with 'Multiple AA entities' providing info to the meeting guide app for the same location.

If two or more entities share meeting information for a given location with the meeting guide app, the app resolves any potential conflicts by only using the info from ONE of these entities. Usually the first one that was linked up with the meeting guide app.

You should consider either asking the GSSA to list the new meeting as well OR work with GSSA and appsupport to have your feed be prioritized for that location

check out [When multiple entities list a meeting](https://meetingguide.helpdocs.io/article/ompwn3xgy7-when-multiple-entities-list-a-meeting)

**Comment by ATLCOTech** [comment]
Hi, 

Thanks for that information. Women 2 Women may not be listed on the Area 
site because they may be a meeting and not a group. Central office lists 
both Area only lists groups. For now I have added "Atlanta" behind Chabad 
Intown as is listed on Google for that organization.

**Comment by SIA-WEBSUP** [comment]
Hey @ATLCOTech,

Are you seeing the meeting appear in the meeting guide app after changing the locationName (and nothing else)? It is my understanding that the app uses the 'de-conflicting' rule I mentioned in my previous post based on the geocoded address and not strictly on the locationName. Therefore, changing  just the locationName might not make  much of a difference. But please let us know if you discover differently!

It would seem to me that if as you say, the area does not list meetings, but your CO does, then this issue you uncovered would occur at any location where a group and a meeting both meet.

IMHO, the best solution would be for you, the area and the app to come to an agreement that for these locations, your feed would be prioritized over the area's.
